export/export_pipeline/compute_gltf_representation.py

```diff
@@ -0,0 +1,270 @@
+      isinstance(actor, unreal.PointLight):
+      continue
+    actor_name = actor.get_actor_label()
+    if 'HDRIBackdrop' in actor_name:
+      continue
+    if include_actors and (actor_name not in include_actors):
+      continue
+    
+    this_body_properties = {'geoms': {}, }
+    # check if actor has geometry
+    has_geometry = False
+    i = 0
+    for c in actor.get_components_by_class(unreal.StaticMeshComponent):
+      static_mesh = c.get_editor_property('static_mesh')
+      if static_mesh is None:
+        print(f'actor {actor_name} component {c.get_name()} does not have a static mesh')
+      elif static_mesh.get_name() != 'SM_Dummy':
+        # TODO(samarth) get MeshComponent elements from UE component
+        c_name = c.get_name()
+        if c_name == 'StaticMeshComponent0':
+          c_name = actor_name
+        this_body_properties['geoms'][c_name] = MeshComponent('coacd', i)._asdict()
+        has_geometry = True
+        i += 1
+
+    if has_geometry:
+      export_meshes(actor, export_dir)
+    else:
+      print(f'Actor {actor_name}, does not have geometry')
+      continue
+    
+    if isinstance(actor, unreal.StaticMeshActor):
+      this_body_properties['moving'] = actor.get_editor_property('static_mesh_component').get_editor_property('body_instance').get_editor_property('simulate_physics')
+    else: # TODO(samarth) get the actual flag for the actuated body
+      this_body_properties['moving'] = False
+    this_body_properties['root_component_name'] = actor.get_editor_property('root_component').get_name()
+    body_properties[actor_name] = this_body_properties
+    
+    constraint_dicts = []
+    for component in actor.get_components_by_class(unreal.ActorComponent):
+      if not isinstance(component, unreal.PhysicsConstraintComponent):
+        continue
+      
+      r = component.get_editor_property('relative_rotation')
+      rot = unreal.Rotator(r.roll, r.pitch, r.yaw)
+      
+      parent = str(component.get_editor_property('component_name2').get_editor_property('component_name'))
+      if (parent == 'DefaultSceneRoot') or (parent == 'StaticMeshComponent0'):
+        parent = actor_name
+      child = str(component.get_editor_property('component_name1').get_editor_property('component_name'))
+      
+      constraint_instance = component.get_editor_property('constraint_instance')
+      angular_offset = constraint_instance.get_editor_property('angular_rotation_offset')
+      constraint_profile = constraint_instance.get_editor_property('profile_instance')
+      
+      limit = 0.0
+      offset = 0.0
+      
+      linear_limit = constraint_profile.get_editor_property('linear_limit')
+      limit = linear_limit.get_editor_property('limit') * TRANS_UE_TO_MUJOCO_SCALE
+      
+      # prismatic joints
+      n_prismatic_dofs = 0
+      pc_args = [parent, child, ]
+      if linear_limit.get_editor_property('x_motion') != unreal.LinearConstraintMotion.LCM_LOCKED:
```

Comment:
I'm skeptical of the code complexity and repetition here. Why don't we just unconditionally export all the relevant joint information (`x_motion`, `y_motion`, `z_motion`, `swing1_motion`, `swing2_motion`, `twist_motion`, etc) and then attempt to construct the appropriate MuJoCo-centric concepts later.